scripts/generate-rncore.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
```

Comment:
can you put short comment explaining what this script does?

Reply:
Will do!